eventually/comment/models.py

```diff
@@ -2,29 +2,49 @@
 Model for describe Comment entity
 """
 
-from django.db import models
-from django.db import IntegrityError
-
+from django.db import models, IntegrityError
+from authentication.models import CustomUser
+from event.models import Event
+from team.models import Team
+from vote.models import Vote
 
 class Comment(models.Model):
     """
-        Comment model that describes comment entity.
+    Comment model that describes comment entity.
+
+    The main idea is to document
+    the class with
+
+    :param text: text of comment
+    :type text: string with max_length = 1024 symbols
+
+    :param created_at: date of comment's create
+    :type  reated_at: datetime
+
+    :param updated_at: date of comment's update
+    :type updated_at: datetime
+
+    :param team: ForeignKey on the certain Team Model
+    :type team: integer
 
-        The main idea is to document
-        the class with
+    :param event: ForeignKey on the certain Event Model
+    :type team: integer
 
-        :param text: text of comment
-        :type text: string with max_length = 1024 symbols
-        :param created_at: date of comment's create
-        :type  reated_at: datetime
-        :param updated_at: date of comment's update
-        :type updated_at: datetime
+    :param vote: ForeignKey on the certain Vote Model
+    :type team: integer
+
+    :param author: ForeignKey on the certain CustomUser Model
+    :type team: integer
 
     """
 
     text = models.CharField(max_length=1024)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
+    team = models.ForeignKey(Team, on_delete=models.CASCADE)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE)
+    vote = models.ForeignKey(Vote, null=True, on_delete=models.CASCADE)
+    author = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
```

Comment:
remove on_delete=models.CASCADE in author

Reply:
done